src/ray/raylet/scheduling/local_resource_manager.h

```diff
@@ -237,7 +246,7 @@ class LocalResourceManager {
   /// Resources of local node.
   NodeResourceInstances local_resources_;
   /// Cached resources, used to compare with newest one in light heartbeat mode.
-  std::unique_ptr<NodeResources> last_report_resources_;
+  mutable std::unique_ptr<NodeResources> last_report_resources_;
```

Comment:
not clear why we change this to mutable.

Reply:
I'll revert it back